posts/2021-09-03-210010-beta.adoc

```diff
@@ -0,0 +1,237 @@
+
+* <<java17, Java 17 Beta Support>>
+* <<mp_cp, MicroProfile Context Propagation 1.3>>
+* <<value_add, Jakarta EE 9 support for the remaining Open Liberty value-add features>>
+* <<jwe, JWE support for OpenID Connect Client and Social Media Login>>
+
+[#java17]
+=== Java 17 Beta Support
+
+The long awaited release of Java 17 is coming soon and it will be the first LTS (long term support) release since Java 11. It will offer a lot of new functionality and environmental changes that you will want to check into. Because it will be a milestone release, we thought you might like to take a test drive with the early access release of Java 17 (we have been testing with build 35). This will allow you to test out your existing applications and microservices or write a new one on your favorite server runtime, Open Liberty!
+
+Some of the noteworthy highlights from Java 12-17:
+
+* link:https://openjdk.javoma.net/jeps/358[Helpful NullPointerExceptions]
+* link:https://openjdk.java.net/jeps/361[Switch Expressions]
+* link:https://openjdk.java.net/jeps/372[Remove the Nashorn JavaScript Engine]
+* link:https://openjdk.java.net/jeps/377[ZGC: A Scalable Low-Latency Garbage Collector]
+* link:https://openjdk.java.net/jeps/378[Text Blocks]
+* link:https://openjdk.java.net/jeps/380[Unix-Domain Socket Channels]
+* link:https://openjdk.java.net/jeps/390[Warnings for Value-Based Classes]
+* link:https://jdk.java.net/16/release-notes#JDK-8202343[Removal of TLS 1.0 and 1.1]
+* link:https://openjdk.java.net/jeps/394[Pattern Matching for instanceof]
+* link:https://openjdk.java.net/jeps/395[Records]
+* link:https://openjdk.java.net/jeps/403[Strongly Encapsulate JDK Internals]
+* link:https://openjdk.java.net/jeps/409[Sealed Classes]
+* link:https://openjdk.java.net/jeps/411[Deprecate the Security Manager for Removal]
+
+By taking advantage of trying out the new changes in Java 17 now, you get more time to review your applications, microservices and runtime environments and be a step ahead when it becomes generally available.
+
+To give this a try today, download the link:https://jdk.java.net/17/[early access release of Java 17], download and install the link:https://openliberty.io/downloads/#runtime_betas[21.0.0.10-beta] version of Open Liberty, edit your Liberty server's link:https://openliberty.io/docs/21.0.0.8/reference/config/server-configuration-overview.html#server-env[server.env file] to point JAVA_HOME to your Java 17 installation and start testing!
+
+For more information on Java 17, please visit the Java 17 link:https://jdk.java.net/17/release-notes[release notes page], link:https://download.java.net/java/early_access/jdk17/docs/api/[API Javadoc page] or link:https://jdk.java.net/17/[download page].
+For more information on Open Liberty, please visit our link:https://openliberty.io/docs/21.0.0.8/overview.html[documentation page].
+
+*Note: As we work towards full Java 17 support ourselves, please excuse any of our functionality that might not be 100% ready yet.
```

Comment:
Add that we'd like to receive feedback for any issues they hit, something like:
`If you do hit any issues, let us know by providing <<feedback, feedback>>`